src/display/text_layer.js

```diff
@@ -637,34 +632,41 @@ var renderTextLayer = (function renderTextLayerClosure() {
         }
         if (expandDivs) {
           transformBuf.length = 0;
-          let padding = '';
+          paddingBuf.length = 0;
 
           if (divProps.angle !== 0) {
             transformBuf.push(`rotate(${divProps.angle}deg)`);
           }
           if (divProps.scale !== 1) {
             transformBuf.push(`scaleX(${divProps.scale})`);
           }
-          if (divProps.paddingLeft > 0) {
-            padding +=
-              ` padding-left: ${divProps.paddingLeft / divProps.scale}px;`;
-            transformBuf.push(
-              `translateX(${-divProps.paddingLeft / divProps.scale}px)`);
-          }
           if (divProps.paddingTop > 0) {
-            padding += ` padding-top: ${divProps.paddingTop}px;`;
+            paddingBuf.push(`${divProps.paddingTop}px`);
             transformBuf.push(`translateY(${-divProps.paddingTop}px)`);
+          } else {
+            paddingBuf.push(0);
           }
           if (divProps.paddingRight > 0) {
-            padding +=
-              ` padding-right: ${divProps.paddingRight / divProps.scale}px;`;
+            paddingBuf.push(`${divProps.paddingRight / divProps.scale}px`);
+          } else {
+            paddingBuf.push(0);
           }
           if (divProps.paddingBottom > 0) {
-            padding += ` padding-bottom: ${divProps.paddingBottom}px;`;
+            paddingBuf.push(`${divProps.paddingBottom}px`);
+          } else {
+            paddingBuf.push(0);
+          }
+          if (divProps.paddingLeft > 0) {
+            paddingBuf.push(`${divProps.paddingLeft / divProps.scale}px`);
+            transformBuf.push(
+              `translateX(${-divProps.paddingLeft / divProps.scale}px)`);
+          } else {
+            paddingBuf.push(0);
           }
 
-          if (padding !== '') {
-            div.setAttribute('style', divProps.style + padding);
+          const padding = paddingBuf.join(' ');
+          if (padding !== NO_PADDING) {
+            div.style.padding = padding;
```

Comment:
Having tested this incrementally on the bot, this line is actually the cause for the Linux/Chrome-only test failure.

It seems that setting `style.padding` directly will truncate some of the last decimals of the padding values, whereas `setAttribute` will keep all the decimals intact (based on checking with the Devtools Inspector).
The difference between keeping all decimals should be *really* small, or even imperceivable, which would explain why most platforms aren't affected by this change. However, it's certainly easy to imagine that the difference (even though it's very small) could trigger ever so slightly different rounding behaviour on some platforms.

All in all, I thus don't think we need to worry about the tiny movement in one of the tests :-)